spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ThriftServiceRegistrationBean.java

```diff
@@ -40,7 +40,7 @@
  * >             .setService(new MyThriftService())
  * >             .setDecorators(LoggingService.newDecorator())
  * >             .setExampleHeaders(ImmutableList.of(HttpHeaders.of(AUTHORIZATION, "bearer b03c4fed1a")))
- * >             .setExampleRequests(ImmutableList.of(new MyThriftService.hello_args("Armeria")))
+ * >             .setExampleRequests(ImmutableList.of(new MyThriftService.hello_args("Armeria")));
```

Comment:
If the above comment is addressed, we could use `addExampleHeaders()` and `addExampleRequests()` here, getting much simpler sample code.